x-pack/plugins/ui_actions_enhanced/public/drilldowns/drilldown_definition.ts

```diff
@@ -20,9 +21,14 @@ import { LicenseType } from '../../../licensing/public';
  * and provided to the `execute` function of the drilldown. This object contains
  * information about the action user performed.
  */
+
 export interface DrilldownDefinition<
   Config extends object = object,
-  ExecutionContext extends object = object
+  SupportedTriggers extends TriggerId = TriggerId,
```

Comment:
Should we just allow triggers in `ActionFactoryDefinition` to be `string[]` instead of complicating these types? Especially if we are planning to remove `TriggerContextMapping` type.

Reply:
I know we discussed that global mapping should go away, but I am not completely imaging the alternative. 
At this point with what we have I like how this all connected. 
It is nice that:
`supported trigger types of drilldown <-> action context` are type checked now based on those global mappings. 

I think till we actually remove a context mapping and have an alternative, we should benefit from it. Maybe we would reconsider removing it before we actually remove it 🤔